app/src/controllers/submission.ts

```diff
@@ -174,7 +196,8 @@ const controller = {
               housingCoopDescription: data.housingCoopName,
               intakeStatus: toTitleCase(data.form.status),
               indigenousDescription: data.IndigenousHousingProviderName,
-              isDevelopedByCompanyOrOrg: toTitleCase(data.isCompany),
+              isDevelopedByCompanyOrOrg:
+                data.isCompany === 'yes' ? ProjectApplicant.BUSINESS : ProjectApplicant.INDIVIDUAL,
```

Comment:
Clarity: I would extend @qhanson55 comment w.r.t. naming clarity. This value appears to be a constrained enumeration type, and not a boolean. Usually the naming style "isSomething" implies a boolean, but this is not the case here. It might be worthwhile to consider having this attribute be more simply named as `projectApplicantType` or something for the following reasons:
- Not a boolean
- Improves semantic flexibility
  - If there's other types coming in the future like corporations or organizations, we can handle this better without the name artificially constraining it.
- Makes it more clear this is a typed value (we have an equivalent ts type definition for this already)